core/src/main/java/org/jobrunr/scheduling/exceptions/InvalidScheduleException.java

```diff
@@ -0,0 +1,8 @@
+package org.jobrunr.scheduling.exceptions;
+
+public class InvalidScheduleException extends RuntimeException {
+
+    public InvalidScheduleException(String message) {
+        super(message);
+    }
```

Comment:
Let's move this to the `Interval` package and make it a bit more smarter: have the message ("Schedule expression cannot be mapped to any type") here as it is only called from one place. Also add the actual expression the user entered and add it to the message - this makes debugging easier.